.github/workflows/release.yaml

```diff
@@ -33,6 +33,7 @@ jobs:
         run: make test
 
   build-and-upload-manifests:
+    if: github.ref_type == 'tag'
```

Comment:
i _think_ this makes sense, but lmk @mitchnielsen @chrisguidry

Reply:
Yeah this looks good, moves the condition from the specific step up to the parent job

Reply:
:fire: